app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("All friends should be vaccinated")
+        elif not visitor["wearing_a_mask"]:

```

Comment:
not clear explanation of the Error. 
```suggestion
            raise OutdatedVaccineError("Vaccine has outdated!")
        elif not visitor["wearing_a_mask"]:

```
Also, instead of ```elif``` use ```if```